Testscripts/Linux/KDUMP-Config.sh

```diff
@@ -6,7 +6,10 @@ kdump_conf=/etc/kdump.conf
 dump_path=/var/crash
 kdump_sysconfig=/etc/sysconfig/kdump
 boot_filepath=""
-
+#Existed bug for kexec-tools https://bugs.launchpad.net/ubuntu/+source/kexec-tools/+bug/1713940
+target_version=2.0.15
+kexec_source_tar="kexec-tools_2.0.15.orig.tar.gz"
```

Comment:
this is the official kexec repo.
https://git.kernel.org/pub/scm/utils/kernel/kexec/kexec-tools.git/

One other issue here - there is no control on the version - now 2.0.15 is good, but in master it's already at 2.0.19.